frontend/src/components/LoginForm.tsx

```diff
@@ -55,7 +56,7 @@ export default function(props: LoginFormProps) {
 
     const loginRequest = useCallback(async () => {
         // Api request here
-        const result = await fetch(`${hostname}/api/user/login`, {method: 'POST', body: JSON.stringify(state), headers: {'Content-Type': 'application/json'} })
```

Comment:
With the new interface this would look like this:
```
const result = await fetch(makeAPIRequest(`/api/user/login`, {method: 'POST', body: JSON.stringify(state), headers: {'Content-Type': 'application/json'} })
```